server/server.go

```diff
@@ -51,7 +52,7 @@ type Info struct {
 	MaxPayload        int      `json:"max_payload"`
 	IP                string   `json:"ip,omitempty"`
 	CID               uint64   `json:"client_id,omitempty"`
-	ClientConnectURLs []string `json:"connect_urls,omitempty"` // Contains URLs a client can connect to.
+	ClientConnectURLs []string `json:"connect_urls,omitempty"` // Contains URLs a client can connect to.Sh
```

Comment:
Remove this?

Reply:
ClientConnectURLs is used by something else - not my code.

Reply:
I am talking about the `Sh` that you have added at the end

Reply:
good catch.